src/main/scala/carldata/series/TimeConverter.scala

```diff
@@ -0,0 +1,91 @@
+package carldata.series
+
+import java.time.LocalDateTime
+
+sealed trait CronElement
+
+case class NumberElement(v: Int) extends CronElement
+
+case class RepeatElement(v: Int) extends CronElement
+
+case class ListElement(s: Seq[Int]) extends CronElement
+
+case class AnyElement() extends CronElement
+
+case class CronLike(minutes: CronElement, hour: CronElement, dayOfMonth: CronElement, month: CronElement
+                    , dayOfWeek: CronElement)
+
+object TimeConverter {
+
+  def mkCronLike(s: String): Either[String, CronLike] = {
+    val xs = s.split(" ")
```

Comment:
IMO it would be better to have the following steps:
 * Split
 * convert to element using parseElement(str: String): Option[CronElement] function
 * if enough parameters then create CronLike structure

As a side note. If the only message in case of error is "cron is incorrect", then it is better to use Option instead of Either. Either is useful if we want to return different different error messages.
We can fix it both ways.